src/realm/array.hpp

```diff
@@ -221,35 +189,18 @@ struct TreeInsertBase {
 /// have a parent. Even if the original was part of a database, the copy will be
 /// free-standing, that is, not be part of any database. For intra, or inter
 /// database copying, one would have to also specify the target allocator.
-class Array : public ArrayParent {
+class Array : public DbElement, public ArrayParent {
 public:
     //    void state_init(int action, QueryState *state);
     //    bool match(int action, size_t index, int64_t value, QueryState *state);
 
     /// Create an array accessor in the unattached state.
-    explicit Array(Allocator&) noexcept;
+    using DbElement::DbElement;
```

Comment:
Unclear. Also, I don't know if this is due to the diff view of github, but it would appear the comment is stale?